core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -537,22 +537,23 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
    */
   private def numExistingExecutors: Int = executorDataMap.size
```

Comment:
how about this one?

Reply:
I intended to add this too, however, all are synchronized when calling this method. I can add it too.

Reply:
then it's ok

Reply:
I have added it...